web-gateway/app/controllers/Main.java

```diff
@@ -38,36 +38,4 @@ public Main(Configuration config, MessagesApi messagesApi, UserService userServi
         );
     }
 
```

Comment:
What about the suggestion to remove the unused fields from `Main`? IDEA should highlight these, too. F2 will jump between problems that IDEA notices. Alt-Enter will provide options to fix them.